src/main/java/ch/njol/skript/expressions/ExprJoinSplit.java

```diff
@@ -92,7 +100,13 @@ public boolean isSingle() {
 	
 	@Override
 	public String toString(final @Nullable Event e, final boolean debug) {
-		return join ? "join " + strings.toString(e, debug) + (delimiter != null ? " with " + delimiter.toString(e, debug) : "") : "split " + strings.toString(e, debug) + (delimiter != null ? " at " + delimiter.toString(e, debug) : "");
+		return join ? "join " + strings.toString(e, debug)
+			+(delimiter != null ? " with "
+			+ delimiter.toString(e, debug) : "")
+			: ((regex ? "regex " : "")
+			+"split " + strings.toString(e, debug)
+			+ (delimiter != null ? " at "
+			+ delimiter.toString(e, debug) : ""));

```

Comment:
This was an unnecessary line change.
```suggestion
		return join ? "join " + strings.toString(e, debug) +(delimiter != null ? " with " + delimiter.toString(e, debug) : "") : ((regex ? "regex " : "") +"split " + strings.toString(e, debug) + (delimiter != null ? " at " + delimiter.toString(e, debug) : ""));

```